contributing/translations/index.rst

```diff
@@ -0,0 +1,90 @@
+Contributing Translations
+=========================
+
+Some Symfony Components include certain messages that must be translated to
+different languages. For example, if a user submits a form with a wrong value in
+a :doc:`TimezoneType </reference/forms/types/timezone>` field, Symfony shows by
+default the following error message: "This value is not a valid timezone."
+
+These messages are translated into tens of languages thanks to the Symfony
+community. Symfony adds new messages on a regular basis, so this is an ongoing
+translation process and you can help us providing the missing translations.
+
+How to Contribute a Translation
+-------------------------------
+
+Imagine that you can speak both English and Swedish and want to check if there's
+some missing Swedish translations to contribute them.
+
+**Step 1.** Translations are contributed to the oldest maintained branch of the
```

Comment:
what if the translation was not added in the oldest branch?

Reply:
Then it's an error of Symfony code (it has happened in the past; we just fixed it). For a translator, this is not important because they won't see the missing translation, so they are not aware that it's missing. They'll see them later when we fix this error and add it.